src/components/NewMessageForm/NewMessageForm.vue

```diff
@@ -883,8 +883,19 @@ export default {
 			})
 		},
 
-		blurInput() {
-			document.activeElement.blur()
+		/**
+		 * Escape was used inside the chat input
+		 *
+		 * Unfocus it when there is no message, otherwise this is used to abort
+		 * autocompletion of emojis and users.
+		 *
+		 * @param {Event} event Key down event
+		 */
+		blurInput(event) {
+			if (this.text.trim() === '') {
```

Comment:
Maybe we should check only what we have either `:`, `:<string>`, `/`, `/<string>`, `@` or `@<string>` as last symbol group in the input value to disable blur?
From current solution, i guess there is no way to blur component by `Esc` if there is something written

Reply:
Was fixed now properly by @ShGKme 

I was trying it the simple way as it's still a better thing then the current behaviour. But now it's properly.